<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menge_dialog_parameter_settings</class>
 <widget class="QDialog" name="menge_dialog_parameter_settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Afbeeldingen/Settings Icon.png</normaloff>:/Afbeeldingen/Settings Icon.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Substeps</string>
   </property>
   <property name="buddy">
    <cstring>spinBox_main_substeps</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum speed pedestrians (m/s)</string>
   </property>
   <property name="buddy">
    <cstring>doubleSpinBox_main_maxspeed</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Time step (s)</string>
   </property>
   <property name="buddy">
    <cstring>doubleSpinBox_main_timestep</cstring>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_main_timestep">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_main_reset">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Values</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pedestrian model</string>
   </property>
   <property name="buddy">
    <cstring>comboBox</cstring>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_main_maxaccel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.500000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>5.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_main_prefspeed">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.300000000000000</double>
   </property>
   <property name="maximum">
    <double>5.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>1.340000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum acceleration pedestrians (m/sÂ²)</string>
   </property>
   <property name="buddy">
    <cstring>doubleSpinBox_main_maxaccel</cstring>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_main_maxspeed">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.500000000000000</double>
   </property>
   <property name="maximum">
    <double>50.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>1.740000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_main_substeps">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mean preferred speed pedestrians (m/s)</string>
   </property>
   <property name="buddy">
    <cstring>doubleSpinBox_main_prefspeed</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Standard deviation preferred speed pedestrians (m/s)</string>
   </property>
   <property name="buddy">
    <cstring>doubleSpinBox_main_prefspeedstddev</cstring>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_main_prefspeedstddev">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>0.500000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.260000000000000</double>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>doubleSpinBox_main_timestep</tabstop>
  <tabstop>spinBox_main_substeps</tabstop>
  <tabstop>doubleSpinBox_main_prefspeed</tabstop>
  <tabstop>doubleSpinBox_main_prefspeedstddev</tabstop>
  <tabstop>doubleSpinBox_main_maxspeed</tabstop>
  <tabstop>doubleSpinBox_main_maxaccel</tabstop>
  <tabstop>pushButton_main_reset</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>menge_dialog_parameter_settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>menge_dialog_parameter_settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
